src/parser/gpx.rs

```diff
@@ -84,11 +236,13 @@ mod tests {
     use xml::reader::EventReader;
     use geo::Point;
 
+    use GpxVersion;
+
     use super::consume;
 
     #[test]
     fn consume_gpx() {
-        let gpx = consume!("<gpx></gpx>");
+        let gpx = consume!("<gpx></gpx>", GpxVersion::Gpx11);
```

Comment:
In retrospect, this should technically error as it lacks a `version=...`.

Reply:
This is will be handled as an error. My updated PR has a bit more test case for this.